docs/source/apis/product.rst

```diff
@@ -0,0 +1,402 @@
+  
+   `GetLowestOfferListingsForASIN <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getlowestofferlistingsforasin>`_ = `get_lowest_offer_listings_for_asin() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L117>`_
+  
+   `GetLowestPricedOffersForSKU <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getlowestpricedoffersforsku>`_ = `get_lowest_priced_offers_for_sku() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L138>`_
+  
+   `GetLowestPricedOffersForASIN <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getlowestpricedoffersforasin>`_ = `get_lowest_priced_offers_for_asin() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L161>`_
+  
+   `GetMyFeesEstimate <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getmyfeesestimate>`_ = `get_my_fees_estimate() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L181>`_
+  
+   `GetMyPriceForSKU <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getmypriceforsku>`_ = `get_my_price_for_sku() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L196>`_
+  
+   `GetMyPriceForASIN <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getmypriceforasin>`_ = `get_my_price_for_asin() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L210>`_
+  
+   `GetProductCategoriesForSKU <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getproductcategoriesforsku>`_ = `get_product_categories_for_sku() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L224>`_
+  
+   `GetProductCategoriesForASIN <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getproductcategoriesforasin>`_ = `get_product_categories_for_asin() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L236>`_
+  
+  
+***************
+Getting Started
+***************
+  
+Please read through gettingStarted.rst, as these docs build upon those.
+
+.. code-block:: python
+
+      import mws
+
+      access_key = “XXXXXXXXXXXXXXXXXXXX”
+      secret_key = “XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX”
+      seller_id = “XXXXXXXXXXXXXX”
+      region = “XX”
+      my_marketplace = Marketplace.US.marketplace_id
```

Comment:
I'm wondering if for emphasis on using the enums for marketplaces, its worth using them each time the mws api is called to hammer this home?

the declaration to a variable is great (DRY!) but for documentation, it may be nicer to highlight this as a way to get the marketplaces as people have a tendency to skip straight to a bit of code that looks like it might be what they want, missing the important and useful bit :)

(I know I do!)